ui/src/infra-model/view/infra-model-view.tsx

```diff
@@ -71,15 +72,12 @@ export type InfraModelViewProps = InfraModelState & {
     onCommitField: (fieldName: string) => void;
 };
 
-const XmlEncodings = ['ISO-8859-1', 'UTF-8', 'UTF-16', 'US ASCII'];
-
-const XmlEncodingsToDropdownEntries = () =>
-    XmlEncodings.map((encoding) => {
-        return {
-            name: encoding,
-            value: encoding,
-        };
-    });
+const xmlEncodingOptions: Item<XmlCharset>[] = [
+    { name: 'ISO-8859-1', value: 'ISO_8859_1' },
+    { name: 'UTF-8', value: 'UTF_8' },
+    { name: 'UTF-16', value: 'UTF_16' },
+    { name: 'US ASCII', value: 'US_ASCII' },
+];
```

Comment:
Tämä lienee se syy minkä takia itse en saanut tuota toimimaan. En itse lähtenyt muuttamaan fronttia vastaamaan bäkkäri-enumien arvoja, vaan olisin itse halunnut vielä saada mäpättyä bäkkärin enumit suoraan noihin wanhoihin stringiarvoihin. Se oli mallia one does not simply. Oikeastaan tämä on ihan hyvä tällä uudella mallilla.